wiki/docs/admins/placeholderapi.md

```
@@ -2,12 +2,179 @@
+
+```yaml
+# ProtocolLib or similar tab formatter
+format: "%player_name% %lumaguilds_rel_%player_name%_status%"
+# Output: Steve🔴 (if at war) or Alex🟢 (if teammate)
+```
+
+## Gotchas
+
+**MiniMessage tags leak in non-MiniMessage plugins:** If you use `%lumaguilds_guild_tag_raw%` in a plugin that doesn't parse MiniMessage (like vanilla TAB), you'll see raw tags like `<color:#FF5733>Elite</color>` in chat. Use `%lumaguilds_guild_tag%` (the legacy variant) instead—it converts MiniMessage to `§`-codes automatically.
+
+**Relation indicator returns empty, not a space:** Neutral relations return a completely blank string (not a space). If you're using this in a player list and want consistent spacing, add a space yourself: `%player_name% %lumaguilds_rel_%player_name%_status% ` (trailing space).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)